lusid-sdk-python/tests/test_aggregation.py

```diff
@@ -0,0 +1,204 @@
+import unittest
+import requests
+import json
+import uuid
+import os
+from datetime import datetime
+import pytz
+import lusid
+import lusid.models as models
+from unittest import TestCase
+from msrest.authentication import BasicTokenAuthentication
+from TestDataUtilities import TestDataUtilities
+from InstrumentLoader import InstrumentLoader
+
+try:
+    # Python 3.x
+    from urllib.request import pathname2url
+except ImportError:
+    # Python 2.7
+    from urllib import pathname2url
+
+
+class TestAggregationFinbourneApi(TestCase):
+    client = None
+    effective_date = datetime(2018, 1, 1, tzinfo=pytz.utc)
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
As we spoke about if you inherit from TestFinbourneApi using from finbournetest import TestFinbourneApi it will do the set up of your client for you. One thing you will have to do though is move anything that is not setting up of the client e.g. getting the instruments from the instrument loader up the top where you have client = None etc. or into a new method such as import_data which you'll need to run first before any of your other methods. 

You should also import timeit from finbournetest and put @timeit above each of your methods. This is a decorator just like @classmethod. What it will do is time how long each of your methods takes and print it to the console. Have a go and see.

There a bunch of other test methods available to you once you inherit from TestFinbourneAPI, you can choose to use all or none of them, might be worth a chat with Riz on this point.